app/serializers/reviewable_category_expert_suggestion_serializer.rb

```diff
@@ -27,8 +26,12 @@ def endorsed_count
     endorsed_users.count
   end
 
-  def category
-    object.target.category
+  def category_id
+    object.target.category_id
+  end
+
+  def include_category_id?
```

Comment:
I think you can add `category_id` to the `attributes` list instead of defining this method

Reply:
no, `category_id` is already added to attributes in the serializer it's inheriting from. `category_id` is also a defined method from the base serializer, and I need to override it